scripts/west_commands/runners/intel_adsp.py

```diff
@@ -69,8 +74,14 @@ def do_add_parser(cls, parser):
         parser.add_argument('--key',
                             help='specify where the signing key is')
         parser.add_argument('--pty',
-                            help='the log will be output immediately to STDOUT, the \
-                            remote log host address is necessary.')
+                            help=''''Capture the output of cavstool.py running on --remote-host \
+                            and stream it remotely to west's standard output.''')
+        parser.add_argument('-o','--run-opt', default=[], action='append',
+                            help='''Additional options for run/request service tool,
+                            e.g. \'--lock\' ''')
+        parser.add_argument('-O', '--log-opt', default=[], action='append',
+                            help='''Additional options for `cavstool`,
+                            e.g. \'--timeout 60\' ''')
```

Comment:
I am confused about this, @enjiamai . You gave a +1 to reserving `-O` for `--tool-opt` here:

https://github.com/zephyrproject-rtos/zephyr/pull/46972#issuecomment-1194509815

But here in the PR you are using `-O` for something different. We can't have both. What's going on?

Reply:
Here is a new pull request where we reserve and standardize on -O for --tool-opt, as previously agreed upon in this PR, for reference:

https://github.com/zephyrproject-rtos/zephyr/pull/48581

Reply:
Thanks! @mbolivar-nordic , working on updating and rebasing this PR, according to your feature extension in the west framework, :)